docs/extending.rst

```diff
@@ -137,22 +160,6 @@ within its ``deserialize`` method.
         if not cstruct:
             return null
 
-Type Constructors
-~~~~~~~~~~~~~~~~~
-
-A type class does not need to implement a constructor (``__init__``),
-but it isn't prevented from doing so if it needs to accept arguments;
-Colander itself doesn't construct any types, only users of Colander
-schemas do, so how types are constructed is beyond the scope of
-Colander itself.
-
-The :exc:`colander.Invalid` exception may be raised during
-serialization or deserialization as necessary for whatever reason the
-type feels appropriate (the inability to serialize or deserialize a
-value being the most common case).
-
-For a more formal definition of a the interface of a type, see
-:class:`colander.interfaces.Type`.
```

Comment:
Nothing from this section has been dropped, only *moved*. See lines 26, 89, 104, and 133.